components/style/properties/longhand/box.mako.rs

```diff
@@ -1279,61 +1243,58 @@ ${helpers.predefined_type("scroll-snap-coordinate",
         }
     }
 
+    // Wrap CSS types for serialization with `write!` or `format!` macros.
+    // Used by ToCss of SpecifiedOperation.
+    struct Css<T>(T);
```

Comment:
Let's move this to `style_traits`, since it may be helpful for other consumers. Also, let's make that comment a doc comment.